R/GroupSequentialDesign.R

```diff
@@ -0,0 +1,103 @@
+#' Group-sequential design
+#'
+#' [ToDo]
+#'
+#' @slot n1 stage-one sample size
+#' @slot c1f early stopping for futility boundary
+#' @slot c1e early stopping for efficacy boundary
+#' @slot n2_pivots stage-two sample size upon continuation
+#' @slot c2_pivots vector of length order giving the values of c2 at the
+#'     pivot points of the numeric integration rule [TODO: these are not available during construction]
+#' @slot x1_norm_pivots normalized pivots for integration rule (in [-1, 1])
+#' @slot weights weights of conditional score values at x1_norm_pivots for
+#'     approximating the integral over x1.
+#'
+#' @exportClass GSDesign
+setClass("GSDesign",  contains = "TwoStageDesign")
```

Comment:
this is crucial - since it is a subclass, it inherits the fields from TwoStageDesign!